server/src/main/java/org/apache/druid/metadata/IndexerSQLMetadataStorageCoordinator.java

```diff
@@ -3022,34 +3012,5 @@ public String getErrorMsg()
     {
       return errorMsg;
     }
-
-    @Override
-    public boolean equals(Object o)
```

Comment:
Any specific reason to clean equals and hashcode ?

Reply:
Instances of this class are never compared for equality.

Reply:
I generally feel that is okay to have this since future developers might use this object in a map or some equality and then that's a bug.

Reply:
I would agree if this was a POJO that was sent over the wire or even passed between classes. But this is meant only for the internal use of `IndexerSqlMetadataStorageCoordinator` and is exposed only for testing.

Better to have someone add it when they actually need it. Also, if someone is using this in a set or as the key of a map, they are doing something very wrong in the first place anyway.